auth_matrix/templates/admin/accounts/user/change_list.html

```diff
@@ -1,27 +1,8 @@
 {% extends "admin/change_list.html" %}
 
-{% block extrastyle %}
-{{ block.super }}
-<style>
-    .extra {
-        padding: 1em;
-    }
+{% load admin_list i18n %}
 
-    .is-flex {
-        display: flex;
-    }
-</style>
-{% endblock extrastyle %}
-
-
-{% block pagination %}
+{% block object-tools-items %}
+    {% include "admin/auth_matrix/includes/object_tools_items.html" %}
```

Comment:
proper namespacing would swap the directory structure: `auth_matrix/admin/includes/object_tools_items.html`